app/i18n/locales/en-US.json

```diff
@@ -39,13 +39,13 @@
   "daedalusTransfer.error.transferFundsError": "Unable to transfer funds.",
   "daedalusTransfer.error.webSocketRestoreError": "Error while restoring blockchain addresses.",
   "daedalusTransfer.errorPage.title.label": "Unable to restore Daedalus wallet.",
-  "daedalusTransfer.form.instructions.step0.text": "Enter the 12-word recovery phrase used to back up your Daedalus wallet to restore the balance and transfer all the funds from Daedalus to Yoroi.",
```

Comment:
Missing update to all of these in other languages (need to be set to the English string)